src/conversion/buildviewconverter.js

```diff
@@ -141,11 +141,21 @@ class ViewConverterBuilder {
 	 * @param {String|RegExp} [value] View attribute value.
 	 * @returns {module:engine/conversion/buildviewconverter~ViewConverterBuilder}
 	 */
-	fromAttribute( key, value = /.*/ ) {
+	fromAttribute( key, value = /.+/ ) {
 		let pattern = {};
-		pattern[ key ] = value;
 
-		return this.from( pattern );
+		if ( key === 'style' || key === 'class' ) {
+			pattern[ key ] = value;
+		} else {
+			pattern.attribute = {};
+			pattern.attribute[ key ] = value;
+		}
+
+		this.from( pattern );
+
+		this._from[ this._from.length - 1 ].attributeKey = key;
```

Comment:
It's not about that. It's about not using `from()` if you actually need to do more than that (and apparently, `from()` is not a sufficient generalisation for this case). `from()` is 5 LOC, just repeat that code here and all will be fine.